src/answers-umd.js

```diff
@@ -428,6 +428,22 @@ class Answers {
       onFinally: config.onFinally || function () {}
     });
   }
+
+  /*
+   * Adds context as a parameter for the query API calls.
+   * @param {Object} context The context object passed in the API calls
+   */
+  setContext (context = {}) {
```

Comment:
I saw this thread in the spec: https://docs.google.com/document/d/1YQTGReyXbrET9FV6_UUZ8xZfYcYLcNQ9W3BtXjbFJpg/edit?disco=AAAAJsIxVwM. Was the decision that the SDK will always have this method and the ability to parse a context from the URL (even if an iframe is not being used)?

Reply:
Yup, the decision is that regardless of its iframed or not, we will parse the `context` query on ANSWERS.init (allowing for people to share context links to people and receive the same experience), and we will also let people `setContext` to dynamically add context through JavaScript.

Reply:
Sounds good!